test_parenthetics.py

```diff
@@ -0,0 +1,24 @@
+"""Test valid parentheses.py."""
+
+import pytest
+
+
+answer_table = [
+    ['', 0],
+    ['(', 1],
+    [')', -1],
+    ['(((((((((())))', 1],
+    ['()()()()((()))', 0],
+    ['()()(((ignorethis)))', 0],
+    ['()()(((ignorethis))))', -1],
+    ['()()(((ignorethis))', 1],
+    [')))(((', -1]
+
```

Comment:
Test the base case of `()`. There are other nested combinations you could've tested but this is pretty good.